base_name_search_improved/tests/test_name_search.py

```diff
@@ -0,0 +1,70 @@
+# © 2016 Daniel Reis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase, at_install, post_install
+
+
+@at_install(False)
+@post_install(True)
+class NameSearchCase(TransactionCase):
+    def setUp(self):
+        super(NameSearchCase, self).setUp()
+        phone_field = self.env.ref("base.field_res_partner_phone")
```

Comment:
FYI, xml id change to field_res_partner__phone 
https://github.com/OCA/server-tools/pull/1419/commits/1374af6011b0093a56adb0171b307be11a5c6771
ping @jjscarafia  Thanks.